templates/zerver/help/zulip-on-android.md

```diff
@@ -1,148 +1,106 @@
 # Using Zulip on Android
 
 Zulip has an Android application, compatible with Android versions 3.2 and
-higher.
-
-It's available on the Google Play Store, and can be downloaded from
-[here](https://play.google.com/store/apps/details?id=com.zulip.android), or by
-clicking the badge below.
-
-[![Play Store badge](/static/images/help/google-play-badge.png)](
-https://play.google.com/store/apps/details?id=com.zulip.android)
+higher. The app can be downloaded from the
+[Google Play Store](https://play.google.com/store/apps/details?id=com.zulip.android).
 
 ## Initial setup
 ### Connecting to a server
 
-When you open the app for the first time, or after you log out, a screen asking
+When you open the app for the first time or after you log out, a screen asking
 for your server's address appears. You have to write the server's URL in it, and
-press *Enter*.
+press **Enter**.
 
-!!! tip ""
-    There's no need to add `https://` in case your server is running secure
+!!! warn ""
+    **Note:** There's no need to add `https://` in case your server is running secure
     HTTP (HTTPS) protocol.
 
-!!! warn ""
-    If your server uses unsecure HTTP, you'll have to
+    However, if your server uses a unsecure HTTP connection, you'll have to
```

Comment:
It's working fine on my side.

![](http://image.prntscr.com/image/3a81272161454448a4a4834a6deabaff.png)